### ISSUE TITLE
Warning in build log

### ISSUE THREAD

**Issue by janvanmansum** [DANS-KNAW/easy-ingest#12]
The build log shows the following warning:

```
[WARNING]  Expected all dependencies to require Scala version: 2.11.7
[WARNING]  nl.knaw.dans.easy:easy-ingest:1.x-SNAPSHOT requires scala version: 2.11.7
[WARNING]  org.json4s:json4s-native_2.11:3.2.11 requires scala version: 2.11.0
[WARNING] Multiple versions of scala libraries detected!
```

Investigate the proper way to resolve this.

**Comment by gkhom** [comment]
This will be resolved when the next "stable" version (3.3.0) of json4s comes out. You can see in the pom.xml of RC6 of 3.3.0 that it depends on Scala 2.11.7 as well ([pom.xml](https://repo1.maven.org/maven2/org/json4s/json4s-native_2.11/3.3.0.RC6/json4s-native_2.11-3.3.0.RC6.pom)).

**Comment by janvanmansum** [comment]
Solved.
